### ISSUE TITLE
No FFB with G27 MAME

### ISSUE THREAD

**Issue by Dookie64** [Boomslangnz/FFBArcadePlugin#69]
I've tried multiple versions of mame, different computers and even a second G27. 

I have followed all the instructions in still get the same results. When I launch MAME, the wheel centers itself but then has zero resistance and no FFB at all in any game I try. My wheel is fully functional in any other driving game I try, mame is the only non responsive program.

I have also watched many videos thinking I missed something but I'm doing everything exactly as told.

**Comment by Boomslangnz** [comment]
Have you added to mame.ini the option to enable outputs ?

**Comment by Dookie64** [comment]
Yes, after I add that file, the wheel centers itself when starting mame.

**Comment by Dookie64** [comment]
> Yes, after I add that file, the wheel centers itself when starting mame.

Sorry, just to clarify. The wheel centers itself with the .ini file but still no FBB in games.

**Comment by jaoxford** [comment]
> Sorry, just to clarify. The wheel centers itself with the .ini file but still no FBB in games.

Which games did you try with specifically?

**Comment by Dookie64** [comment]
Most supported MAME games, all the rush games, crusin usa/world, dirt dash etc.

**Comment by leonardo201800478** [comment]
There is a problem here with MAME 0.276, in the GUI tests the FFB works, the steering wheel centers when starting the game, but nothing really works in the game, it stopped working in the new versions of MAME.
